pages/gallery.html

```diff
@@ -4,9 +4,41 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link rel="stylesheet" href="../stylesheets/style.css">
+  <link rel="stylesheet" href="../stylesheets/gallery.css">
+  <title>Gallery</title>
 </head>
 <body>
-  
+  <header>
+    <h1> Real Pictures of the Blue Ridge Mountains</h1>
+    <nav>
+      <ul>
+        <li><a href="index.html">Home</a></li>
+        <li><a href="facts.html">Cold Hard Facts</a></li>
+        <li><a href="recreation.html">Recreation</a></li>
+        <li><a href="gallery.html">Gallery</a></li>
+    </ul>
+    </nav>
+  </header>
+  <!--Hiking Pics-->
+  <section class="hiking-mod">
+    <h2>Hiking Trails</h2>
```

Comment:
It's a little subjective, but in this layout I might consider the header and the section as part of the significant content of the page and place it inside a `main` tag.